# Sensor Model Parameters for a Spherical Depth Sensor Model
num_rings: 64
num_ring_points: 360
num_basis_shapes: 9
noise_std_dev: 0.03

# Forward speed params
forward_speed: 1.5
forward_speed_max: 2.0
forward_speed_obst_gain: 0.5

# Lateral speed params
lateral_speed_gain: -1.0
lateral_speed_max: 2.5

# Vertical speed params
vertical_speed_gain: -1.0
vertical_speed_max: 2.0

# Yaw rate params
yaw_rate_gain: -1.0
yaw_rate_max: 0.6

# Lateral Dynamic Controller Parameters
Mv_size: 6
Mv_A: [0.9981, 0, 0, 0, 0, 0, 2.0412, 0.0528, -0.2010, -0.2612, -0.4296,
      -7.7748, 6.4777, 0.0270, -0.1694, -1.0669, -1.2556, -24.6354, 5.3906,
      -0.0436, 0.1789, 0.3458, -0.8861, -20.4777, 0.9735, -0.0128, 0.0638,
      0.2929, 0.8476, -3.6970, 0.0036, -0.0001, 0.0003, 0.0021, 0.0119, 0.9862]
Mv_B: [0.0999, 0.5016, 0.6738, 0.2434, 0.0291, 0.0001]
Mv_C: [282.8, -10.7, -14.6, -21.0, -38.1, -1073.3]

# Heading Dynamic Controller Parameters
Mr_A: [0.9999, 0.0000, 0.0000, 0.0000, 4.1504, 0.2350, -0.3828, -10.5831,
      3.0972, -0.1106, 0.5083, -7.9036, 0.0187, -0.0008, 0.0072, 0.9524]
Mr_B: [0.0200, 0.0582, 0.0373, 0.0001]
Mr_C: [139.2399, -8.7243, -12.4138, -355.0898]

# Vertical Dynamic Controller Parameters
Mw_A: [0.9996, 0.0000, 0.0000, 0.0000, 1.7728, 0.1060, -0.1395, -3.3989,
      6.3121, -0.3233, 0.3998, -12.1131, 0.0550, -0.0053, 0.0078, 0.8945]
Mw_B: [0.1000, 0.2087, 0.4397, 0.0022]
Mw_C: [78.6214, -17.2273, -4.9615, -150.7364]


 